test/helpers/position_assertions.cc

```diff
@@ -1010,7 +1010,6 @@ void CompletionAssertion::check(const UnorderedMap<string, shared_ptr<core::File
     auto completionList = doTextDocumentCompletion(wrapper, *this->range, nextId, this->filename);
     ASSERT_NE(completionList, nullptr) << "doTextDocumentCompletion failed; see error above.";
 
-    // TODO(jez) Add ability to expect CompletionItemKind of each item
```

Comment:
I don't think this is that important to test; it just affects which icon shows up.